pkg/cmd/get/experiments.go

```diff
@@ -75,39 +77,63 @@ var experimentsCmd = &cobra.Command{
 			}
 		}
 
-		output, err := cmd.Flags().GetString("output")
+		outputFormat := ""
+		outputPrompt := promptui.Select{
+			Label: "Select output format",
+			Items: []string{"Table", "json", "yaml"},
```

Comment:
better to follow smallcase for string constants, `Table` can be changed to `table`